### PR TITLE
Handled settings json read write.

### PR DESCRIPTION
**Pull request by Shivraj0**
[JIRA](https://gofynd.atlassian.net/browse/FPCO-1921)

### CODE REVIEW

File: src/lib/Theme.ts [hunk 1]
```diff
@@ -53,6 +53,37 @@ export default class Theme {
...
     static SRC_ARCHIVE_FOLDER = './.fdk/archive';
     static ZIP_FILE_NAME = `archive.zip`;
+    static SETTINGS_DATA_PATH = path.join(process.cwd(), '/theme/config/settings_data.json');
+    static SETTINGS_SCHEMA_PATH = path.join(process.cwd(), '/theme/config/settings_schema.json');
+
+    public static async writeSettingJson(path, options) {
+        try {
+            await fs.writeJSON(
+                path,
+                options,
```
Comment (vivek8690) [inline src/lib/Theme.ts]: change options to jsonOject because it contains JSON object. @Shivraj0

Comment (Shivraj0) [reply src/lib/Theme.ts]: Done @vivek8690

File: src/lib/Theme.ts [hunk 2]
```diff
@@ -53,6 +53,37 @@ export default class Theme {
     static SRC_FOLDER = './.fdk/temp-theme';
     static SRC_ARCHIVE_FOLDER = './.fdk/archive';
     static ZIP_FILE_NAME = `archive.zip`;
+    static SETTINGS_DATA_PATH = path.join(process.cwd(), '/theme/config/settings_data.json');
```
Comment (vivek8690) [inline src/lib/Theme.ts]: global variable won't work @Shivraj0

Comment (Shivraj0) [reply src/lib/Theme.ts]: Done @vivek8690

**Pull request merged**